diff --git a/fs/namespace.c b/fs/namespace.c
index fd999ca..ad00c8b 100644
--- a/fs/namespace.c
+++ b/fs/namespace.c
@@ -31,6 +31,7 @@
 
 /* spinlock for vfsmount related operations, inplace of dcache_lock */
 __cacheline_aligned_in_smp DEFINE_SPINLOCK(vfsmount_lock);
+EXPORT_SYMBOL_GPL(vfsmount_lock);
 
 static int event;
 
@@ -1877,3 +1878,4 @@ void __put_mnt_ns(struct mnt_namespace *ns)
 	release_mounts(&umount_list);
 	kfree(ns);
 }
+EXPORT_SYMBOL_GPL(__put_mnt_ns);
