Index: linux-2.6.18/fs/namespace.c
===================================================================
--- linux-2.6.18.orig/fs/namespace.c	2006-02-18 14:36:04.000000000 -0600
+++ linux-2.6.18/fs/namespace.c	2006-02-18 14:37:37.000000000 -0600
@@ -45,7 +45,8 @@
 static struct list_head *mount_hashtable __read_mostly;
 static int hash_mask __read_mostly, hash_bits __read_mostly;
 static kmem_cache_t *mnt_cache __read_mostly;
-static struct rw_semaphore namespace_sem;
+struct rw_semaphore namespace_sem;
+EXPORT_SYMBOL_GPL(namespace_sem);
 
 /* /sys/fs */
 decl_subsys(fs, NULL, NULL);
Index: linux-2.6.18/include/linux/namespace.h
===================================================================
--- linux-2.6.18.orig/include/linux/namespace.h	2006-02-18 14:35:49.000000000 -0600
+++ linux-2.6.18/include/linux/namespace.h	2006-02-18 14:37:55.000000000 -0600
@@ -13,6 +13,7 @@
 	int event;
 };
 
+extern struct rw_semaphore namespace_sem;
 extern int copy_namespace(int, struct task_struct *);
 extern void __put_namespace(struct namespace *namespace);
 extern struct namespace *dup_namespace(struct task_struct *, struct fs_struct *);
